canonical-web-and-design/vanilla-framework
# Add demo of class reference to side navigation component

## Done

Improves initial demo of class reference docs for side navigation.
While we are not going to immediately start adding them for other components, it would be good not to waste this piece of work.

## QA

- Open [demo](https://vanilla-framework-4328.demos.haus/docs/patterns/navigation#class-reference)
- Check the "Class reference" section in side navigation docs
  - https://vanilla-framework-4328.demos.haus/docs/patterns/navigation#class-reference
- Make sure that it works well on different screen sizes
- Read the descriptions to check for any errors or grammar issues.

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)


## Screenshots

<img width="898" alt="Screenshot 2022-02-21 at 13 04 49" src="https://user-images.githubusercontent.com/83575/154952024-bed092b2-bc4c-43a7-8b81-4c6362b401cb.png">
